runtime/vm/ValueTypeHelpers.cpp

```diff
@@ -25,8 +25,11 @@
 #include "j9.h"
 #include "ut_j9vm.h"
 #include "ObjectAccessBarrierAPI.hpp"
+#include "vm_api.h"
 
 extern "C" {
+#define J9_ARRAY_ELEMENT_SIZE(arrayClass) (arrayClass->flattenedElementSize)
```

Comment:
make this a real helper, like the others

Reply:
Do I keep the macro naming convention too?

Reply:
you can call it arrayElementSize(..)